sensor_msgs/Image img
string action
int64[] matrix